config/routes.rb

```diff
@@ -21,12 +21,15 @@
 
     get '/eligibility' => 'start_page#eligibility', as: :eligibility
     post '/eligibility' => 'start_page#determine_eligibility'
+    get '/not_eligible', to: 'start_page#not_eligible', as: :not_eligible
 
     get '/sign-up', to: 'sign_up#new', as: :sign_up
     post '/sign-up', to: 'sign_up#create'
+    get '/check-email/sign-up', to: 'sign_in#check_your_email', as: :check_email_sign_up
```

Comment:
To me this makes more sense, sign-up is the "top-level" thing and "check-email" is a part of sign-up.

```suggestion
    get '/sign-up/check-email', to: 'sign_in#check_your_email', as: :sign_up_check_email
```

Reply:
I was following the pattern from the prototype, but probably makes more sense to use `/sign-up/check-email` 👍

Reply:
Resolved in dafafa3